src/input/accessors.cc

```diff
@@ -142,8 +142,8 @@ std::string Address::make_full_address() const {
         } else
 		if (typeid(*input_type) == typeid(Type::AbstractRecord)) {
 			const Type::AbstractRecord * a_rec = static_cast<const Type::AbstractRecord *>(input_type);
-			const StorageInt * storage_type = static_cast<const StorageInt *>(storage->get_item(0));
-			input_type = & a_rec->get_descendant((unsigned int)storage_type->get_int());
+			const StorageString * storage_type = static_cast<const StorageString *>(storage->get_item(0));
```

Comment:
type() is method of accessor Input::AbstractRecord, but in this case we have Input::Type::AbstractRecord